Amplify/Categories/API/AmplifyAPICategory+APICategory.swift

```diff
@@ -19,6 +19,15 @@ extension AmplifyAPICategory: APICategory {
             throw error
         }
 
+        guard !isConfigured else {
+            let pluginDescription = String(describing: plugin)
+            let error = ConfigurationError.amplifyAlreadyConfigured(
+                "\(pluginDescription) has already been configured.",
+                "Remove the duplicate call to `Amplify.configure().`"
```

Comment:
Here and throughout: error message doesn't match the condition
```suggestion
                "Do not add plugins after calling `Amplify.configure().`"
```